C/c4Document.cc

```diff
@@ -110,28 +108,24 @@ C4Document::~C4Document() {
 
 
 C4Document::C4Document(const C4Document &doc)
-:_flags(doc._flags)
-,_docID(doc._docID)
-,_revID(doc._revID)
-,_sequence(doc._sequence)
-,_selected(doc._selected)
-,_selectedRevID(doc._selectedRevID)
-,_collection(doc._collection)
-{
+        : _flags(doc._flags), _docID(doc._docID), _revID(doc._revID), _sequence(doc._sequence), _selected(doc._selected),
```

Comment:
Shoving all the initializers together makes them much harder to read. If these were assignment statements, they'd be on separate lines, right? Initializers are very important, including the order they happen in, so they should be as clear as possible.

Reply:
I think CLion's formatter was doing something weird with indents. Luckily it comes with built-in Clang-Format support, so I've switched to that